packages/server/graphql/mutations/helpers/addSeedTasks.ts

```diff
@@ -1,27 +1,41 @@
 import convertToTaskContent from 'parabol-client/utils/draftjs/convertToTaskContent'
 import getTagsFromEntityMap from 'parabol-client/utils/draftjs/getTagsFromEntityMap'
+import makeAppURL from 'parabol-client/utils/makeAppURL'
+import appOrigin from '../../../appOrigin'
 import getRethink from '../../../database/rethinkDriver'
 import {RValue} from '../../../database/stricterR'
 import {TaskStatusEnum} from '../../../database/types/Task'
 import generateUID from '../../../generateUID'
+import {convertHtmlToTaskContent} from '../../../utils/draftjs/convertHtmlToTaskContent'
 
-const CONTENT_STRING = `
-  This is a task card. They can be created here, in a meeting, or via an integration`
+const NORMAL_TASK_STRING = `This is a task card. They can be created here, in a meeting, or via an integration`
+const INTEGRATIONS_TASK_STRING = `Parabol supports integrations for Jira, GitHub, GitLab, Slack and Mattermost. Connect your tools on Settings > Integrations.`
 
-const SEED_TASKS = [
-  {
-    status: 'active' as TaskStatusEnum,
-    sortOrder: 0,
-    content: convertToTaskContent(CONTENT_STRING),
-    plaintextContent: CONTENT_STRING
-  }
-]
+function getSeedTasks(teamId: string) {
+  const integrationURL = makeAppURL(appOrigin, `team/${teamId}/settings`)
```

Comment:
-1 we should go to `/settings/integrations` directly

Reply:
I checked and there is no `/settings/integrations` route?

Reply:
@JimmyLv This will forward user directly to integrations tab `team/${teamId}/settings/integrations`

Reply:
aha! I got it!  no worries😆